### PR TITLE
Dont add start build scripts

### PR DESCRIPTION
**Pull request by vsavkin**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: docs/generated/packages/react/generators/application.json [hunk 1]
```diff
@@ -159,11 +159,6 @@
         "enum": ["babel", "swc"],
         "default": "babel"
       },
-      "skipDefaultProject": {
```
Comment (FrozenPandaz) [inline docs/generated/packages/react/generators/application.json:162]: Keep it as a flag so commands don't fail but add `x-deprecated`.

File: packages/nx/src/adapter/ngcli-adapter.ts [hunk 2]
```diff
@@ -1222,7 +1222,6 @@ function saveWorkspaceConfigurationInWrappedSchematic(
   );
   nxJson.generators = workspace.generators || workspace.schematics;
   nxJson.cli = workspace.cli || nxJson.cli;
-  nxJson.defaultProject = workspace.defaultProject;
```
Comment (FrozenPandaz) [inline packages/nx/src/adapter/ngcli-adapter.ts:1225]: I'm not sure if this should be removed. If someone writes this into `angular.json`, we should move it over to `nx.json`

File: packages/react/src/generators/application/schema.json [hunk 3]
```diff
@@ -165,11 +165,6 @@
       "enum": ["babel", "swc"],
       "default": "babel"
     },
-    "skipDefaultProject": {
```
Comment (FrozenPandaz) [inline packages/react/src/generators/application/schema.json:168]: Keep it as a flag so commands don't fail but add `x-deprecated`.

File: packages/workspace/src/generators/preset/preset.ts [hunk 4]
```diff
@@ -140,12 +140,6 @@ async function createPreset(tree: Tree, options: Schema) {
   }
 }
 
-function addPolyfills(host: Tree, polyfillsPath: string, polyfills: string[]) {
```
Comment (FrozenPandaz) [inline packages/workspace/src/generators/preset/preset.ts:143]: This seems unrelated. Why was this removed?

File: packages/workspace/src/migrations/update-13-0-0/config-locations/config-locations.spec.ts [hunk 5]
```diff
@@ -91,7 +90,6 @@ describe('update to v13 config locations', () => {
       });
       expect(workspaceJson.projects.a.tags).toEqual(['test']);
       expect(workspaceJson.cli).not.toBeDefined();
-      expect(workspaceJson.defaultProject).not.toBeDefined();
```
Comment (FrozenPandaz) [inline packages/workspace/src/migrations/update-13-0-0/config-locations/config-locations.spec.ts:94]: Again, I think if they have it, we should move it over.

**Review by FrozenPandaz — CHANGES_REQUESTED** [review]

File: packages/workspace/src/generators/preset/preset.ts (hunk 4 above)
Comment (vsavkin) [reply packages/workspace/src/generators/preset/preset.ts:143]: Unused

File: packages/react/src/generators/application/schema.json (hunk 3 above)
Comment (vsavkin) [reply packages/react/src/generators/application/schema.json:168]: Fixed

File: packages/workspace/src/migrations/update-13-0-0/config-locations/config-locations.spec.ts (hunk 5 above)
Comment (vsavkin) [reply packages/workspace/src/migrations/update-13-0-0/config-locations/config-locations.spec.ts:94]: Migration test
